.pnpmfile.cjs

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  hooks: {
+    readPackage: (pkg) => {
+      if (pkg.name === 'zx') {
+        // zx use "@types/node": ">=20" as optionalDependencies, which may bring some unexpected updates in other packages
+        delete pkg.optionalDependencies['@types/node'];
```

Comment:
Could we use https://github.com/pnpm/pnpm/releases#:~:text=Added%20the%20ability%20for%20overrides%20to%20remove%20dependencies%20by%20specifying%20%22%2D%22%20as%20the%20field%20value%20%238572.%20For%20example%2C%20to%20remove%20lodash%20from%20the%20dependencies%2C%20use%20this%20configuration%20in%20package.json%3A as it's more convenient.

Reply:
it only deal with `dependencies`, here is `optionalDependecies`

Reply:
😂 Nevermind, we're good to merge.